core/src/test/scala/integration/kafka/api/SaslSslAdminClientIntegrationTest.scala

```diff
@@ -119,4 +150,126 @@ class SaslSslAdminClientIntegrationTest extends AdminClientIntegrationTest with
     assertFutureExceptionTypeEquals(results.results().get(clusterAcl), classOf[InvalidRequestException])
     assertFutureExceptionTypeEquals(results.results().get(emptyResourceNameAcl), classOf[InvalidRequestException])
   }
+
+  val FOO_ACL = new AclBinding(new Resource(ResourceType.TOPIC, "foobar"),
+    new AccessControlEntry("User:ANONYMOUS", "*", AclOperation.READ, AclPermissionType.ALLOW))
+  val FOO_ACL_FILTER = FOO_ACL.toFilter
```

Comment:
These are just vals in a class, so should be `fooAcl`. If they were in an object, they would be `FooAcl`. Also, generally, it's a good idea to place all the vals at the top of the file to avoid weird NPEs due to initialisation order.